OpenRA.Mods.Common/SpriteLoaders/GenieSlpLoader.cs

```diff
@@ -54,14 +80,14 @@ static byte[] ReadRowCommands(Stream stream, GenieSlpFrameHeader header)
 					return rowData.ToArray();
 				}
 
-				var opcode = 0xf;
-				var rowPosX = 0;
+				int opcode;
```

Comment:
This gets set 5 lines down, before it's ever used, so you were setting it twice.